changes-set.txt

```diff
@@ -99,9 +99,10 @@ make a github issue.)
 
 DONE:
 Date      Old       New         Notes
-20-Aug-24 brsnop    [same]	moved from TA's mathbox to main set.mm
+21-Aug-24 2exp11    [same]	    moved from AV's mathbox to main set.mm
+20-Aug-24 brsnop    [same]	    moved from TA's mathbox to main set.mm
 19-Aug-24 dfrp2     [same]      moved from TA's mathbox to main set.mm
-19-Aug-24 rabeqcda  [same]	moved from SN's mathbox to main set.mm
+19-Aug-24 rabeqcda  [same]	    moved from SN's mathbox to main set.mm
```

Comment:
Why are the entries indented?

Reply:
they are indented because in various editors, a tab displays as less than 8 spaces. Here, it seems the tab was displayed as four spaces so 4 spaces were added to "match".

the tabs are replaced with spaces in my pr: https://github.com/metamath/set.mm/pull/4149

It might be useful to check for tabs in changes-set.txt

Reply:
Yeah exactly, in vscode it looked unaligned to me and I put a tab to align it. After I pushed I saw that it looks weird in github. Honestly this is why Google's coding style guidelines forbid tabs and only allow spaces as whitespace characters. But since @icecream17 will fix it in his pr I will leave it as is.